xcodeproj/internal/processed_target.bzl

```
@@ -0,0 +1,153 @@
+""" Functions for processing Xcode Targets."""
+
+load(
+    ":product.bzl",
+    "product_to_dto",
+)
```

Since this doesn't need to wrap anymore, can it be one line?
```suggestion
load(":product.bzl", "product_to_dto")
```